examples/deep_dream.py

```diff
@@ -83,8 +83,8 @@ def deprocess_image(x):
 loss = K.variable(0.)
 for layer_name in settings['features']:
     # Add the L2 norm of the features of a layer to the loss.
-    assert (layer_name in layer_dict.keys(),
-            'Layer ' + layer_name + ' not found in model.')
+    assert layer_name in layer_dict, ('Layer ' + layer_name + 
```

Comment:
Please replace the assert with a ValueError. It will lead to a better UX and it will make the code clearer (e.g. preventing the previous issue).